addons/binding/org.openhab.binding.dwdunwetter/src/main/java/org/openhab/binding/dwdunwetter/internal/DwdUnwetterConfiguration.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.dwdunwetter.internal;
+
+import org.apache.commons.lang.StringUtils;
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ * The {@link DwdUnwetterConfiguration} class contains fields mapping thing configuration parameters.
+ *
+ * @author Martin Koehler - Initial contribution
+ */
+public class DwdUnwetterConfiguration {
```

Comment:
I think the default values are already used from the `config-description` in the thing xml and can be removed completely.

Reply:
I removed than from the member definition, but for the return I like to keep them to not get a NPE if something went wrong.